static/panes/compiler.ts

```diff
@@ -987,7 +988,7 @@ export class Compiler extends MonacoPane<monaco.editor.IStandaloneCodeEditor, Co
 
     popAndRevealJump(): void {
         if (this.revealJumpStack.length > 0) {
-            this.editor.restoreViewState(this.revealJumpStack.pop() as any);
+            this.editor.restoreViewState(this.revealJumpStack.pop() ?? null);
```

Comment:
`restoreViewState` can take an `ICodeEditorViewState | null` but `this.revealJumpStack.pop()` is `ICodeEditorViewState | null | undefined`, so this is the easiest way I can think of to make the types consistent